testcontainers-dapr/src/main/java/io/dapr/testcontainers/Configuration.java

```diff
@@ -19,20 +19,24 @@
 public class Configuration {
   private final String name;
   private final TracingConfigurationSettings tracing;
+  private final AppHttpPipeline appHttpPipeline;
 
-  //@TODO: add httpPipeline
-  //@TODO: add secrets
-  //@TODO: add components
-  //@TODO: add accessControl
+  // @TODO: add secrets
```

Comment:
Would you be open to opening issues to add support for these TODO items here for general awareness & visibility?

Reply:
done